src/main/java/me/contaria/seedqueue/SeedQueueEntry.java

```diff
@@ -44,6 +44,7 @@ public class SeedQueueEntry {
     private volatile boolean locked;
     private volatile boolean loaded;
     private volatile boolean discarded;
+    private volatile boolean prioritized = true;

```

Comment:
Please rename `SeedQueueEntry#prioritized` to something more descriptive along the lines of `maxWorldGenerationReached`.

Reply:
Makes sense yeah